source/docs/hardware/hardware-basics/robot-battery.rst

```diff
@@ -42,47 +39,44 @@ Battery Construction & Tools
+The Anderson SB Connector may be the standard Pink/Red SB50, or another Anderson SB connector. Teams are *STRONGLY* recommended to use the Pink/Red SB50 for interoperability: the other colors and sizes of housings will not intermate, and you will be unable to borrow batteries or chargers.
 
-Terminals for the battery tabs are available online and through electrical supply houses, sold by the accepted wire size and the tab hole diameter, sometimes listed on catalogs as a "post diameter".
+Follow manufacturer's instructions to crimp contacts and assemble the leads into Anderson SB connectors. A small flathead screwdriver can help to insert the contacts (carefully push on the contact, not the wire insulation), or it can help to disengage the internal latch if the contact is in the wrong slot or upside down.
 
-Teams are strongly recommended to use either use properly rated tools and stringent quality control practices for the crimping process, or use vendor-made crimped leads.
+Bolt-on crimp lugs with 0.2" tab holes (compatible with #10 bolt) battery tabs are available online and through electrical supply houses, sold by the accepted wire size and the tab hole diameter, sometimes listed on catalogs as a "post diameter". Use a size that matches the lead wires.
 
-Bolt-on screw terminals are not recommended for this connection.
+Teams are strongly recommended to use properly rated tools and stringent quality control practices for crimping processes, ask local veteran teams for help, or use vendor-made crimped leads. *Bolt-on screw terminal lugs are not recommended for battery leads.*
 
-Battery Connections
-^^^^^^^^^^^^^^^^^^^^
+Battery Lug Connection
+^^^^^^^^^^^^^^^^^^^^^^^
 
-The terminals must not move after they are torqued (tightened).
-It is a good idea to secure the lugs with a real lock washer, such as a #10 star washer or a nordlock system, in addition to a nylon locking nut.
-Some batteries come with bolts in the package. They may be replaced with stronger alloy steel bolts.
+A #10 or M5 nut & bolt connect the battery lead lug to the battery tab.
 
-Using a split ring lock washer will not add to joint strength or reliability.
+..warning:: The lug and tab must directly contact, copper to copper: do not put a washer of any kind separating them.
 
-..warning:: The lug and tab must be in direct contact, copper to copper: do not put a washer of any kind separating them.
+These connections must be very tight for reliability. Any movement of the lug while in operation may interrupt robot power, resulting in robot reboots and disconnections lasting 30 seconds or more.
+
+Some batteries come with tab bolts in the package: they may be used, or replaced with stronger alloy steel bolts. It is a good idea to add a lock washer, such as a #10 star washer or a nordlock system, in addition to a nylon locking ("nylock") nut. Using a split ring washer will not add to joint strength or reliability and is not required even if one is provided with your battery.
```

Comment:
>Using a split ring lock washer will not add...

Standing here in MechE land, split ring "lock" washers are the devil's fastener. They are shown in research papers to help joints fall apart under vibration.

Yet in the "common troubleshooting" article, we say "use the split ring lock washer at the PDP main lug"? Which I suppose is helpful in making sure students actually tighten the connection, rather than thinking the Fancy Electrical Thing is too fancy to get torqued down hard.